buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -388,6 +388,30 @@ public static long swapLong(long value) {
     }
 
     /**
+     * Writes a big-endian 16-bit short integer to the buffer.
+     */
+    @SuppressWarnings("deprecation")
+    public static ByteBuf writeShortBE(ByteBuf buf, int shortValue) {
+        return buf.order() == ByteOrder.BIG_ENDIAN? buf.writeShort(shortValue) : buf.writeShortLE(shortValue);
```

Comment:
@Scottmitch I use code style from this guide: http://netty.io/wiki/setting-up-development-environment.html